README.md

```diff
@@ -1,6 +1,6 @@
 # README
 
-This script adds starred GitHub repositories as bookmarks in Pinboard. All such bookmarks will be tagged with `github-star` and the project's language (if applicable). The script will attempt to avoid overwriting bookmarks in Pinboard by checking starred repos against the most recent bookmark with the `github-star` tag.
+This script adds starred GitHub repositories as bookmarks in Pinboard. All such bookmarks will be tagged with `github-star` and the project's language (if applicable). The script will attempt to avoid overwriting bookmarks in Pinboard by checking starred repos against the most recent bookmark with the `github-star` tag. If the GitHub repository has a listed language the language will be added as another tag in addition `github-star`, you may set a prefix to the language tag with the `--lang-tag-prefix`.
```

Comment:
I know it's a bit nitpicky :) but how about changing this paragraph to:

This script adds starred GitHub repositories as bookmarks in Pinboard. All such bookmarks will be tagged with `github-star` and the project's language (if applicable). In addition, you may set a prefix for the language tag with the `--lang-tag-prefix` option. The script will attempt to avoid overwriting bookmarks in Pinboard by checking starred repos against the most recent bookmark with the `github-star` tag.